nncf/torch/hardware/fused_patterns.py

```diff
@@ -37,6 +42,8 @@ def _get_torch_hw_fused_patterns() -> HWFusedPatterns:
 
     retval.register(linear_ops + batch_norm_activations_permutation, 'LINEAR + BN_ACT_PERM',
                     match=True)
+    retval.register(matmul_ops + arithmetic_ops, 'MATMUL + ARITHMETIC',
+                    match=True)
```

Comment:
Could you please add a [synthetic model](https://github.com/openvinotoolkit/nncf/blob/5d60171af4b8a4bf840a9af0b5074c8b11d2d5a0/tests/torch/test_models/synthetic.py#L196) with this pattern and add a new test case here in the [test_synthetic_model_quantization](https://github.com/openvinotoolkit/nncf/blob/develop/tests/torch/test_compressed_graph.py#L718)?

Reply:
Done, see the .dot file visualization with the pattern fused:
![Screenshot from 2021-09-30 18-01-56](https://user-images.githubusercontent.com/31695470/135481240-4c6298cb-3ec3-4f99-8307-7a5af153f624.png)